media/css/protocol/components/send-to-device.scss

```diff
@@ -7,7 +7,7 @@ $image-path: '/media/protocol/img';
 
 @import '../../../protocol/css/includes/lib';
 
-.send-to-device {
+#send-to-device, .send-to-device {
```

Comment:
There still seems to be some broken styling on other pages that's being introduced by these changes:

(Left: this branch, Right: prod)

![send-to-device](https://user-images.githubusercontent.com/400117/68590120-af834a80-0485-11ea-8240-a769b3a43f41.png)

Reply:
there are now two different classes the default is horizontal and the is a new class for vertical

Reply:
there are now two different classes the default is horizontal and the is a new class for vertical.